conda/cli/main_list.py

```diff
@@ -217,7 +217,8 @@ def execute(args, parser):
     if args.canonical:
         format = 'canonical'
     elif args.export:
-        format = 'export'
+        print_explicit(prefix, args.md5)
```

Comment:
This makes `conda list --export` and `conda list --explicit` exactly the same. The former is supposed to be different, and separated by `=`.